.gitignore

```diff
@@ -1,2 +1,17 @@
-.DS_Store
-.idea
\ No newline at end of file
+*
```

Comment:
why you need these? if you use modman you don't have to put plugin within magento dir

Reply:
It was there. It ignores all files except Bolt module files.